nucypher/blockchain/eth/actors.py

```diff
@@ -358,10 +347,80 @@ def publish_aggregated_transcript(
             public_key=public_key,
             participant_public_key=participant_public_key,
             transacting_power=self.transacting_power,
-            fire_and_forget=True,
         )
         return tx_hash
 
+    def get_phase_receipt(
```

Comment:
What are the phases?

Reply:
It's another name for the dkg "rounds" while trying to avoid name shadowing `round` in several places.